packages/framework-integration-tests/integration/provider-unaware/end-to-end/cart.integration.ts

```diff
@@ -377,9 +375,6 @@ describe('Cart end-to-end tests', () => {
           `,
         })
 
-        console.log(`Waiting ${secs} second${secs > 1 ? 's' : ''} for deletion to complete...`)
-        await sleep(secs * 1000)
-
```

Comment:
why this is no longer needed?

Reply:
This was introduced back in 872cf430, probably due to network or lag issues that were making the tests fail at the time. Considering that integration tests pass without this `sleep`, we can confidently remove it.